.golangci-kal.yml

```diff
@@ -25,7 +25,7 @@ linters-settings:
           
           # Linters below this line are disabled, pending conversation on how and when to enable them.
           # - "commentstart" # Ensure comments start with the serialized version of the field name.
-          # - "integers" # Ensure only int32 and int64 are used for integers.
+          - "integers" # Ensure only int32 and int64 are used for integers.
```

Comment:
Can you please move this up to the top where conditions is? Otherwise the comment on L#26 doesn't make sense